training-service/test/hack3rz_test.py

```diff
@@ -90,8 +90,9 @@ def load_test_X_T(self, lang_name):
             training data array X and target data array T. For more informationn check information in src/services/training.py
         
         """
-        X, T = data_preprocessing(self.annotation_repository.find_data_to_train_with(lang_name))
-        return X, T
+        training_data = self.annotation_repository.find_data_to_train_with(lang_name)
```

Comment:
Mmmmh, since we removed the data_preprocessing() in this function it is exactly the same than a "normal" database call. It only made sense previously because of the two test, i.e. fetch data from database and preprocess it. I think we don't need the whole function anymore because we simply can execute ```self.annotation_repository.find_data_to_train_with(lang_name)``` in our tests as it is already done at the moment. 

TODO:
- Remove this function and replace its calls in the tests with ```self.annotation_repository.find_data_to_train_with(lang_name)```directly